### PR TITLE
Docs for dataset runs index and show routes

### PR DESCRIPTION
**Pull request by marianmosley**
For https://github.com/mode/issues/issues/16204

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -1894,7 +1894,56 @@ paths:
         required: true
         type: string
   "/{account}/reports/{report}/runs":
-    x-summary: Report Runs
+    x-summary: Report Run
```
Comment (monteverdetico) [inline swagger.yaml:1897]: Should this still be `Report Runs`?

**Review by monteverdetico — COMMENTED** [review]
One question, but otherwise looks good.

File: swagger.yaml (hunk 1 above)
Comment (marianmosley) [reply swagger.yaml:1897]: This is for the `POST`, which returns a run, so I think this is ok.

Comment (monteverdetico) [reply swagger.yaml:1897]: Ah, yes, correct. :)

Comment (monteverdetico) [reply swagger.yaml:1897]: Or, wait, on line 1898 it says `get:` -- this file is hard to read so not sure if it is for the above or for something below.

**Review by monteverdetico — APPROVED** [review]

**Pull request merged**